CMakeLists.txt

```diff
@@ -33,270 +80,229 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-      VS_STARTUP_PROJECT ${PROJECT_NAME}
-  )
-
-  # set Visual Studio working directory to the Configuration directory
-  set_target_properties(
-    ${PROJECT_NAME}
-    PROPERTIES
-      VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/$(Configuration)"
-  )
-endif()
+if (DOXYGEN_FOUND)
+    set(DoxyFile_src ${PROJECT_SOURCE_DIR}/Doxyfile)
+    set(DoxyFile ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
+    configure_file(${DoxyFile_src} ${DoxyFile} @ONLY)
+
+    message("Use target doc to build the documentation")
+    add_custom_target(
+            doc
+            COMMAND ${DOXYGEN_EXECUTABLE} ${DoxyFile}
+            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+            VERBATIM
+    )
+endif ()
+
+if (MSVC)
+    # set startup project for Visual Studio Builds
+    set_property(
+            DIRECTORY ${PROJECT_SOURCE_DIR}
+            PROPERTY
+            VS_STARTUP_PROJECT ${PROJECT_NAME}
+    )
+    # set Visual Studio working directory to the Configuration directory
+    set_target_properties(
+            ${PROJECT_NAME}
+            PROPERTIES
+            VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/$(Configuration)"
+    )
+endif ()
 
 install(TARGETS ${PROJECT_NAME} DESTINATION .)
-if ( NOT APPLE )
-  install(DIRECTORY resources DESTINATION .)
+if (NOT APPLE)
+    install(DIRECTORY resources DESTINATION .)
 
-  if ( EXISTS ${PROJECT_SOURCE_DIR}/Cytopia_Resources )
-    install(DIRECTORY ${PROJECT_SOURCE_DIR}/Cytopia_Resources/music DESTINATION ./resources/audio)
-  endif()
-endif()
+    if (EXISTS ${PROJECT_SOURCE_DIR}/Cytopia_Resources)
+        install(DIRECTORY ${PROJECT_SOURCE_DIR}/Cytopia_Resources/music DESTINATION ./resources/audio)
+    endif ()
+endif ()
 
 install(FILES ReleaseNotes.txt credits.txt DESTINATION .)
 
-
-#on windows, we need to deploy the SDL2 dynamic libraries
-if ( WIN32 )
-  set (SDL_RUNTIME_LIBRARIES
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libFLAC-8.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libfreetype-6.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libjpeg-9.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libmodplug-1.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libmpg123-0.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libogg-0.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libopus-0.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libopusfile-0.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libpng16-16.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libtiff-5.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libvorbis-0.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libvorbisfile-3.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/libwebp-7.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/SDL2.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/SDL2_image.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/SDL2_mixer.dll
-  ${SDL2_RUNTIME_LIBRARY_DIR}/SDL2_ttf.dll
-  ${ZLIB_RUNTIME_LIBRARY}
-  )
-
-  # copy the qt libraries over to the build directory so the tool can be started
-  add_custom_command(
-    TARGET ${PROJECT_NAME} POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different
-    ${SDL_RUNTIME_LIBRARIES}
-    $<TARGET_FILE_DIR:${PROJECT_NAME}>
-  )
-
```

Comment:
Can't test it with the current PR, but this will still be necessary if building without package manager, since the libs are linked dynamically per default.

Do you want to add static linking or rather add that back? (or add an option to decide if it should be linked static or dynamically, but that might be too much...)